### PR TITLE
Datetime control component

### PR DESCRIPTION
**Pull request by k0nG**
TP: https://rockabox.tpondemand.com/entity/9085

- Using same date time directive from rbx_ui: http://mgcrea.github.io/angular-strap/#/datepickers
- Pulling custom bootstrap css generated by http://getbootstrap.com/customize/ to just include things we need to datepicker.
- Including extra datepicker css from https://github.com/mgcrea/bootstrap-additions/blob/master/src/datepicker/datepicker.less
- Added `eot` file loading to webpack.
- Changed `grunt test` task to export `TZ=UTC` environmental var so phantomJs runs in UTC timezone and therefore produces predictable results when testing time inputs.
- Added help text on input to tell user that time presented in in their local browser time.

### CODE REVIEW

File: src/rb-datetime-control/rb-datetime-control.css [hunk 1]
```diff
@@ -0,0 +1,1709 @@
...
+  font-weight: bold;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+td,
+th {
+  padding: 0;
+}
```
Comment (nwhite89) [inline src/rb-datetime-control/rb-datetime-control.css]: I think everything between this line and Line 18 should be removed as it's going to override our own normalise stuff added by @ohskylab

File: src/rb-datetime-control/rb-datetime-control.tpl.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        ng-disabled="{{isDisabled}}"
+        ng-readonly="{{isReadonly}}"
+        ng-class="{'is-invalid': form['date'].$touched && form['date'].$invalid}"
+        placeholder="{{placeholderDate}}"
+        autocomplete="off"
+        bs-datepicker>
+    <input
+        type="text"
+        class="TextControl-field"
+        name="time"
```
Comment (nwhite89) [inline src/rb-datetime-control/rb-datetime-control.tpl.html]: Doesn't this make it so you can't have multiple date time's for the same form?

File: src/rb-datetime-control/demo/demo.tpl.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        <h2>
+            rb-datetime-control
+        </h2>
+        <h3>
+            Datetime Control Angular Version
+        </h3>
+    </div>
+    <div class="ComponentView">
+        <h4>Basic usage</h4>
+        <form name="basicForm">
```
Comment (nwhite89) [inline src/rb-datetime-control/demo/demo.tpl.html]: Can you use one form for the demo, to ensure that you can have multiple date time field's on one form please

File: Gruntfile.js [hunk 4]
```diff
@@ -12,12 +12,18 @@ module.exports = function (grunt) {
...
         var tasks = [
             'lint',
+            'env:testUTC',
             'karma:unit'
         ];
 
         grunt.task.run(tasks);
     });
 
+    grunt.registerTask('unit', [
```
Comment (nwhite89) [inline Gruntfile.js]: can you update the README please

File: src/rb-datetime-control/rb-datetime-control.css (hunk 1 above)
Comment (k0nG) [inline src/rb-datetime-control/rb-datetime-control.css]: Yeah agreed. Will remove as much as I can.

File: Gruntfile.js (hunk 4 above)
Comment (k0nG) [inline Gruntfile.js]: Done.

File: dev/app.js [hunk 5]
```diff
@@ -36,6 +37,7 @@ define([
     rbButtonDemo,
     rbCurrencyDemo,
     rbDataSummaryDemo,
+    rbDatetimeControl,
```
Comment (chrisvxd) [inline dev/app.js]: @k0nG this is the `Demo`

**Comment by nwhite89** [comment]
kl I'm happy for merge would like @chrisvxd to sign off also as added more comment's than me

**Pull request merged**